README.md

```diff
@@ -303,13 +311,36 @@ when exploring classes and methods.
 
 Autocompletion currently cannot be disabled.
 
+![suggestions](https://user-images.githubusercontent.com/79095619/213680153-38f5df12-8304-480b-aae2-37e6df1433f2.gif)
+
+
+## Modification of the selected lines
+
+Sometimes it is useful to perform some operation on the selected lines.
+The CodeEditor exposes an API for that purpose through the `modifySelectedLines` method inside `CodeController`
+
+```dart
+controller.modifySelectedLines((line) {
+  return line.toUpperCase(); // performs modification on each line
+});
+```
```

Comment:
I am not sure of this API. We can later change how `indent`, `outdent`, and uncommenting are implemented. Let's make it `@internal` and not advertise it.